migrate-to-chimera

```diff
@@ -124,6 +100,36 @@ for u in * ; do
     fi
 done
 
+# Check if there are config files from steam-buddy
+if [ -d $HOME/.config/steam-buddy ] ; then
+    if [ -d $HOME/.config/chimera ] ; then
+        cp -Rbv $HOME/.config/steam-buddy/* $HOME/.config/chimera/.
+        rm -rfv $HOME/.config/steam-buddy
+        echo "Configuration directories have been merged and existing chimera files been backed up"
+    else
+        mv $HOME/.config/steam-buddy $HOME/.config/chimera
+        echo "Configuration directory has been moved"
+    fi
+fi
+
+# Migrate cache
+if [ -d $HOME/.cache/steam-buddy ] ; then
+    # Remove chimera cache if exists
+    if [ -d $HOME/.cache/chimera ] ; then
+        rm -rf $HOME/.cache/chimera
+    fi
+    echo "Moving cache"
+    mv $HOME/.cache/steam-buddy $HOME/.cache/chimera
+fi
+
+# Migrate user data directories and files from steam-buddy
+if [ -d $HOME/.local/share/steam-buddy ] ; then
+    if [ -d $HOME/.local/share/chimera ] ; then
+        rm -rfv $HOME/.local/share/chimera
```

Comment:
I don't think we should be removing things. This could potentially be a legit directory the user has created.

Reply:
Fair enough, we can move everything inside `~/.local/share/steam-buddy/*` to `~/.local/share/chimera` we could also backup automatically if it's needed with `mv -b`